lib/firefox/webdriver/firefoxPreferences.js

```diff
@@ -11,38 +11,10 @@ module.exports = {
   'dom.performance.time_to_first_interactive.enabled': true,
   // Use hidden TTCP
   'dom.performance.time_to_contentful_paint.enabled': true,
-  // IPV6 sometimes makes DNS slow on Linux
-  'network.dns.disableIPv6': true,
   'browser.startup.homepage': 'about:blank',
-  /* Extra tracking protection https://wiki.mozilla.org/Security/Tracking_protection */
-  'privacy.trackingprotection.enabled': false,
-  'privacy.trackingprotection.pbmode.enabled': false,
-  'privacy.trackingprotection.annotate_channels': false,
-  'services.sync.prefs.sync.privacy.trackingprotection.enabled': false,
-  'services.sync.prefs.sync.privacy.trackingprotection.pbmode.enabled': false,
-  'browser.safebrowsing.provider.mozilla.updateURL': '',
-  'browser.safebrowsing.provider.mozilla.gethashURL': '',
-
-  /* Disable heartbeat https://wiki.mozilla.org/Firefox/Shield/Heartbeat */
-  'browser.selfsupport.url': '',
-  'browser.selfsupport.enabled': false,
 
   'devtools.devedition.promo.enabled': false,
 
-  /* Disable detectportal.firefox.com */
-  'network.captive-portal-service.enabled': false,
-  'browser.search.geoip.url': '',
-  /* Disable telemetry  https://wiki.mozilla.org/Telemetry/Testing */
-  'toolkit.telemetry.prompted': 2,
-  'toolkit.telemetry.rejected': true,
-  'toolkit.telemetry.enabled': false,
-  'toolkit.telemetry.reportingpolicy.firstRun': false,
```

Comment:
I'm worried that we'll start submitting a bunch of bogus Telemetry, but that's only a concern in automation.  Automation will do its own things to profiles to avoid this, so roll on.

Reply:
Good point.  I can set "toolkit.telemetry.server" to "https://127.0.0.1/telemetry-dummy/" like in https://searchfox.org/mozilla-central/source/testing/profiles/perf/user.js#93